vendor/github.com/hashicorp/serf/serf/serf.go

```diff
@@ -1654,8 +1654,6 @@ func (s *Serf) Stats() map[string]string {
 	toString := func(v uint64) string {
 		return strconv.FormatUint(v, 10)
 	}
-	s.memberLock.RLock()
```

Comment:
This undid this monkey patch from https://github.com/hashicorp/serf/pull/486, which I just merged. Can you re-up and vendor again?

Reply:
fixed